explorer/interpreter/interpreter.cpp

```diff
@@ -1150,10 +1221,8 @@ auto Interpreter::StepExp() -> ErrorOr<Success> {
       } else {
```

Comment:
```
  bool interface_member = access.impl().has_value() && !forming_member_name;
  ...
      // Would normally be 2, but because the condition above is `act.pos() == 1 && foo`, we will have to approach that differently
      } else if (act.pos() == 1 && interface_member || act.pos() == 2) {
        // ... Instantiate type
      } else {
        // Get result from instantiate type, likely with `act.results()[n]` here, not sure at which index it is stored here
        return todo_FinishAction(act.results()[0]);
      }
        
      }
```